### PR TITLE
Fix broken links/images

### PR DESCRIPTION
**Pull request by Dhaulagiri**
Update paths on images and links to be relative to root so that they can
be found

### CODE REVIEW

**Comment by trek** [comment]
Don't they really need to be relative to location, not root?

**Comment by trek** [comment]
e.g.

instead of 
```diff
```

**Comment by trek** [comment]
e.g.

instead of 
```diff
-Element](../components/customizing-a-components-element).
+Element](../../components/customizing-a-components-element).
```

Does
```diff
-Element](../components/customizing-a-components-element).
+Element](../customizing-a-components-element).
```
also work?

**Comment by rwjblue** [comment]
The issue here is Divshot's pretty urls.

Specifically, from `http://guides.emberjs.com/v1.10.0/components/customizing-a-components-element/` (note the trailing slash) to get to `http://guides.emberjs.com/v1.10.0/templates/binding-element-attributes` the relative path is `../../templates/binding-element-attributes`.  If however the source location was `http://guides.emberjs.com/v1.10.0/components/customizing-a-components-element` the relative path would be `../templates/binding-element-attributes`.

File: source/components/defining-a-component.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Each component, under the hood, is backed by an element. By default
 Ember will use a `<div>` element to contain your component's template.
 To learn how to change the element Ember uses for your component, see
 [Customizing a Component's
-Element](../components/customizing-a-components-element).
+Element](../../components/customizing-a-components-element).
```
Comment (rwjblue) [inline source/components/defining-a-component.md]: should use `./customizing-a-components-element` instead

File: source/cookbook/helpers_and_components/index.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 Here are some recipes to help you encapsulate your code into Components and build Helpers.
 
-1. [Creating Reusable Social Share Buttons](/guides/cookbook/helpers_and_components/creating_reusable_social_share_buttons)
-2. [A Spinning Button for Asynchronous Actions](/guides/cookbook/helpers_and_components/spin_button_for_asynchronous_actions)
-3. [Adding Google Analytics Tracking](/guides/cookbook/helpers_and_components/adding_google_analytics_tracking)
+1. [Creating Reusable Social Share Buttons](../../cookbook/helpers_and_components/creating_reusable_social_share_buttons)
```
Comment (rwjblue) [inline source/cookbook/helpers_and_components/index.md]: Should be `./helpers_and_components/creating_reusable_social_share_buttons`.

File: source/cookbook/helpers_and_components/index.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
...
+2. [A Spinning Button for Asynchronous Actions](../../cookbook/helpers_and_components/spin_button_for_asynchronous_actions)
```
Comment (rwjblue) [inline source/cookbook/helpers_and_components/index.md]: Should be `./spin_button_for_asynchronous_actions`.

File: source/components/defining-a-component.md (hunk 1 above)
Comment (Dhaulagiri) [inline source/components/defining-a-component.md]: This does not work (at least locally), but `../customizing-a-components-element` does work

**Comment by Dhaulagiri** [comment]
@trek has a point.  I could go back and find the links that can be relative to their location and update those accordingly.

File: source/components/defining-a-component.md (hunk 1 above)
Comment (rwjblue) [inline source/components/defining-a-component.md]: :+1:

**Comment by trek** [comment]
Please do.

**Comment by Dhaulagiri** [comment]
My latest commit should have addressed that @trek.  Can you review?

**Comment by trek** [comment]
awesome.

**Comment by Dhaulagiri** [comment]
Let me know when you deploy and I can spot check some things to make sure everything looks ok

**Comment by trek** [comment]
You changes are deployed to http://development.ember-guides.divshot.io/v1.10.0/

Once you give us the :+1: I can promote that build to production.

**Comment by Dhaulagiri** [comment]
:+1: looks good from the items I spot checked @trek.

**Comment by trek** [comment]
Great. I promoted development to production.

**Comment by rwjblue** [comment]
:heart:

**Comment by Dhaulagiri** [comment]
Nice.  If I find other related problems I can open a new issue for those.

**Pull request merged**